src/lib.rs

```diff
@@ -48,8 +48,10 @@ pub async fn run() -> Result<()> {
         ui.println(
                 r#"Welcome to the Amazon QLDB Shell!
 
-To start a transaction type 'start transaction', after which you may enter a series of PartiQL statements.
-When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
+- Transactions: By default, the shell implicitly runs each statement in its own transaction and automatically commits the transaction if no errors are found. This is configurable. To start a multi-statement transaction, enter `start transaction` or `begin`.
+- PartiQL: a subset of PartiQL is supported, and elements returned by queries are in unordered "bags." For more details, check the shell guide in the developer guide.
```

Comment:
Suggesting minor copyedits to clarify and convert this to active voice rather than passive:

>PartiQL: QLDB supports a subset of PartiQL, and returns elements of query results in unordered "bags". For more details, see the shell topic in the QLDB Developer Guide.

Reply:
Thanks, updated